privval/signer_listener_endpoint.go

```diff
@@ -186,6 +196,12 @@ func (sl *SignerListenerEndpoint) serviceLoop() {
 			{
 				conn, err := sl.acceptNewConnection()
 				if err == nil {
+					remoteAddr := conn.RemoteAddr()
+					if !sl.isAllowedAddr(remoteAddr) {
+						sl.Logger.Info(fmt.Sprintf("deny a connection request from remote address=%s", remoteAddr))
```

Comment:
I think it is also advisable to log which address is allowed to connect from, for problem analysis in case of actual rejection.

Reply:
Thanks, It would be more descriptive.